projects/packages/videopress/src/client/admin/components/input/style.module.scss

```
@@ -0,0 +1,24 @@
+.wrapper {
+	width: 100%;
+	height: calc( var( --spacing-base ) * 5 ); // 40px;
+	padding: var( --spacing-base );
+	border: 1px solid var( --jp-black );
+	border-radius: 4px;
```

```suggestion
        border-radius: var(--jp-border-radius);
```

You can find all the available vars here: https://github.com/Automattic/jetpack/blob/trunk/projects/js-packages/components/components/theme-provider/index.tsx